pkg/controller/machine/machine_controller_suite_test.go

```diff
@@ -65,10 +65,18 @@ func StartTestManager(mgr manager.Manager, t *testing.T) chan struct{} {
 	t.Helper()
 
 	stop := make(chan struct{})
+	errs := make(chan error, 1)
+
 	go func() {
 		if err := mgr.Start(stop); err != nil {
-			t.Fatalf("error starting test manager: %v", err)
+			errs <- err
 		}
 	}()
+
+	err := <-errs
```

Comment:
This will block until the channel receives anything (which in most cases would be never cause `err == nil` is normal?). Might be best to return this error channel as an extra param and then assign it to a var that can be checked in the `AfterEach` once `stop` has been closed, WDYT?

Reply:
good idea